### PR TITLE
Use variables for input border width

### PR DESCRIPTION
**Pull request by marcoambrosini**
https://github.com/nextcloud-libraries/nextcloud-vue/assets/26852655/a2c1e554-3488-4d99-8b82-a3c377f768ab

### CODE REVIEW

**Comment by ShGKme** [comment]
> Waiting for server `border-width` variable to be in before merging this

Can you link the PR? I cannot find it...

**Comment by marcoambrosini** [comment]
@ShGKme it doesn't exist yet :)

**Review by jancborchardt — CHANGES_REQUESTED** [review]
The input field looks good, but for the textarea, it jumps around vertically on focus, a result of the 1px/2px change on focus.

(And yes, waiting for server for the --border-width variable.)

**Comment by marcoambrosini** [comment]
variable here https://github.com/nextcloud/server/pull/45247/commits/43510be0343aaedce183950e7a4a555951ec00aa

File: src/components/NcInputField/NcInputField.vue [hunk 1]
```diff
@@ -371,7 +371,7 @@ export default {
 
 		background-color: var(--color-main-background);
 		color: var(--color-main-text);
-		border: 2px solid var(--color-border-maxcontrast);
+		border: var(--border-width-input) solid var(--color-border-maxcontrast);
```
Comment (ShGKme) [inline src/components/NcInputField/NcInputField.vue:374]:
```suggestion
		border: var(--border-width-input, 2px) solid var(--color-border-maxcontrast);
```

Comment (ShGKme) [reply src/components/NcInputField/NcInputField.vue:374]: And the same in other places

**Review by ShGKme — COMMENTED** [review]
For compatibility with older server, we probably should use 2px as a fallback

**Comment by marcoambrosini** [comment]
@ShGKme I've switched back to borders, but there's still a problem with the text jumping inside on hover and focus. I've tried to change the paddings but the problem is still there. Could you please have a look?

**Review by susnux — APPROVED** [review]

**Review by szaimen — APPROVED** [review]
🐘

**Comment by ShGKme** [comment]
> but there's still a problem with the text jumping inside on hover and focus

Was it resolved?

**Pull request merged**